lib/cli/commands/serve.js

```diff
@@ -25,6 +25,18 @@ serve.builder = function(cli) {
 			default: false,
 			type: "boolean"
 		})
+		.option("show-hidden", {
+			describe: "Show hidden files in the served index.",
+			default: false,
```

Comment:
As stated in https://github.com/SAP/ui5-server/pull/255#discussion_r341977950, we would rather have a default `true` here as that is the current behavior.